docs/examples-profiling-vtune.md

```diff
@@ -0,0 +1,80 @@
+
+After profiling is complete, a results folder will hold profiling data that can then be read and analyzed with VTune.
+
+Also note, VTune is capable of profiling a single process or system wide. As such, and like perf, VTune is plenty capable of profiling the wasmtime runtime itself without any added support. However, the APIs [`here`](https://github.com/intel/ittapi) also support an interface for marking the start and stop of code regions for easy isolatation in the VTune Profiler. Support for these APIs are expected to be added in the future.
+
+For example, with VTune properly installed, if you're using the CLI you'll execute with:
+
+```sh
+$ cargo build --features=vtune
+$ amplxe-cl -run-pass-thru=--no-altstack -collect hotspots target/debug/wasmtime --vtune foo.wasm
+```
+
+This command tells the collector (amplxe-cl) to collect hotspot profiling data on wasmtime running foo.wasm. The --vtune flag enables VTune support in wasmtime so that the collector is also alerted to jit events that take place during runtime. The first time this is run, the result of the command is a results diretory r000hs/ which contains hotspot profiling data for wasmtime and the execution of foo.wasm. This data can then be read and displayed via the command line or via the VTune gui by importing the result.
+
+### `VTune` example
+
+Running through a familiar algorithm, first we'll start with the following wasm:
+
+```rust
+fn main() {
+    let n = 45;
+    println!("fib({}) = {}", n, fib(n));
+}
+
+fn fib(n: u32) -> u32 {
+    if n <= 2 {
+        1
+    } else {
+        fib(n - 1) + fib(n - 2)
+    }
+}
+```
+
+Profiling data using vtune can be collected a number of ways and profiling data can be collected to focus
+on certain types of analysis. Below we show a command line executable option using amplxe-cl, which is
+installed and in our path, to help find hotspots in our wasm module. To collect  profiling information then,
+we'll simply execute:
+
+```sh
+$ rustc --target wasm32-wasi fib.rs -C opt-level=z -C lto=yes
+$ amplxe-cl -run-pass-thru=--no-altstack -v -collect hotspots target/debug/wasmtime --vtune fib.wasm
+fib(45) = 1134903170
+amplxe: Collection stopped.
+amplxe: Using result path \`/home//jlb6740/wasmtime/r000hs\'
+amplxe: Executing actions  7 % Clearing the database
+amplxe: The database has been cleared, elapsed time is 0.239 seconds.
+amplxe: Executing actions 14 % Updating precomputed scalar metrics
+amplxe: Raw data has been loaded to the database, elapsed time is 0.792 seconds.
+amplxe: Executing actions 19 % Processing profile metrics and debug information
+...
+...
+Top Hotspots
+Function                                                                                      Module          CPU Time
+--------------------------------------------------------------------------------------------  --------------  --------
+h2bacf53cb3845acf                                                                             [Dynamic code]    3.480s
```

Comment:
Oh dear I see what you mean, it looks like some mangling has definitely gone awry here by accident :(

Reply:
Let me just remove this.